hera_mc/tests/test_correlator.py

```diff
@@ -362,7 +362,13 @@ def test_config_errors(self):
     def test_add_correlator_config_from_corrcm_onsite(self):
 
         result = self.test_session.add_correlator_config_from_corrcm(testing=True)
+
         self.assertTrue(len(result) > 0)
+        if len(result) == 2:
```

Comment:
Would we ever get more than 2 results?

Reply:
No. We can only get 2 at most, that happens if there's not a corresponding file already known about, so we need to make an entry in the correlator_config_file table as well as an entry in the correlator_config_status table.